example/entropy_analysis.md

```diff
@@ -1,12 +1,12 @@
 ```markdown
```

Comment:
coud you remove the ```markdown line? Previously, it gave problems with correctly rendering the page, and I had to remove it manually

Reply:
All looking good on the website. No more action required from me?

Reply:
No, indeed. Thanks.